### ISSUE TITLE
Using the package on Windows

### ISSUE THREAD

**Issue by asigalov61** [SgtPooki/npm-bulk-import-cid#3]
Hey @SgtPooki ,

I was able to install and run this on my local machine (Windows 11) but it does not seem to be doing anything...

It installs the package fine, but the daemon does not appear to start:

All I see in the console is this:

`Need to install the following packages:
@sgtpooki/bulk-import-cid@1.0.1
Ok to proceed? (y) y
`

I use your example.txt for texting so there are 3 CIDS in that file.

Can you help me make it work? 

Thank you very much.

Alex.

**Comment by asigalov61** [comment]
@SgtPooki 

Also, in Google Colab, the daemon appears to start but it seems that colab restricts the connection needed (UDP).

`Starting up Kubo daemon...
Reading CIDs from file at path /content/test.txt
Read 1 lines from file
daemon stdout: Initializing daemon...
daemon stdout: 
daemon stdout: Kubo version: 0.23.0
daemon stdout: Repo version: 15
daemon stdout: System version: amd64/linux
daemon stdout: Golang version: go1.21.1
daemon stdout: 
daemon stderr: 2024/02/03 08:47:08 failed to sufficiently increase receive buffer size (was: 208 kiB, wanted: 2048 kiB, got: 416 kiB). See https://github.com/quic-go/quic-go/wiki/UDP-Buffer-Sizes for details.
daemon stderr: 
Shutting down...`

**Comment by asigalov61** [comment]
@SgtPooki A little update...

I got it working on Windows 11. The trick was to add kubo to Windows PATH vars (both system and user) and also I had to run npx package from CMD as an administrator.

MAKE SURE TO RESTART THOROUGHLY so that the paths vars can update on windows.

So it seems to be working great!!! Thank you so much!

Still does not work in Google Colab though and it would be nice to see it integrated into web-ui soon :)

Again, thank you for your great work :)

Alex
